Sensory/src/main/java/com/google/android/sensory/sensing_sdk/db/impl/dao/ResourceInfoDao.kt

```diff
@@ -27,7 +27,7 @@ import com.google.android.sensory.sensing_sdk.model.ResourceInfo
 
 @Dao
 internal abstract class ResourceInfoDao {
-  @Insert(onConflict = OnConflictStrategy.ABORT)
+  @Insert(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
If this fixes a problem you have I'd suggest it's a sign that something else is seriously wrong.

Are you using insert to replace something?

Reply:
So essentially this was already in mind while designing I just missed this part.

What we want to do is that the capture API should be able to re-capture the data for the same participant. For that we do 2 things:-
1. We first delete everything in the last capture (Changes in CaptureViewModel)
2. We REPLACE the ResourceInfoEntity if there is a conflict (by unique folder path)

What we can definitely do is improve this by not deleting last captured data until new data is captured [But that is ok I guess]